### ISSUE TITLE
Court order page refinements

### ISSUE THREAD

**Issue by JimKillock** [openrightsgroup/blocked-org-uk#138]
1 Group the blocks together by URL, that is all the ISP blocks for a single URL consecutively.

2 Remove duplicates caused by case errors, eg Www. and www.

3 Remove URLs that include information after / eg

piratebay.org/user/pirate

*unless* the domain itself is not blocked, or not in the database.

4 Include a statistic: Number of domains detected as blocked by court order

**Comment by dantheta** [comment]
Sort order is done.  The ordering of url blocks is done according to the maximum first_block date, so as new blocks are discovered on new networks urls will be pulled to the top of the list.

**Comment by JimKillock** [comment]
Thanks Dan :) I've amended (3) and added a (4), to display number of domains blocked.

**Comment by dantheta** [comment]
A bunch of duplicates have been cleaned out.  The URL normalization function needs a bit of work to stop more creeping in.

**Comment by dantheta** [comment]
site count added

**Comment by JimKillock** [comment]
Interesting duplicate URL added today: Http://thepiratebay.org

**Comment by dantheta** [comment]
Yep, mobile platforms automatically capitalize the first character of most input fields.

**Comment by dantheta** [comment]
1,2 and 4 are done. Would suggest punting 3 to post launch, since it's data cleanup that isn't hugely user visible.

**Comment by dantheta** [comment]
Moving (3) to a separate issue.
